src/Installer/redist-installer/targets/GenerateLayout.targets

```diff
@@ -12,7 +12,6 @@
     <AspNetCoreBlobVersion>$(MicrosoftAspNetCoreAppRefInternalPackageVersion)</AspNetCoreBlobVersion>
 
     <NetRuntimeBlobVersion>$(VSRedistCommonNetCoreSharedFrameworkx6490PackageVersion)</NetRuntimeBlobVersion>
```

Comment:
I don't think this is produced in source build. I think you want to use something like...Microsoft.NETCore.Platforms

Reply:
It's explicitly set here: https://github.com/dotnet/sdk/blob/71ca55412ce46d108ad929c79a5c3c48adea0662/eng/Versions.props#L106

Why wouldn't that be available from SB? What property from there is preferred?

Reply:
I think it will work for the official build but a dev/ci SB would fail because VS.* packages aren't produced. Not as much of a concern for .NET 9 but UB will definitely need to be correct here. I recommend MicrosoftNETCorePlatformsPackageVersion. We can use that in MS, UB, SB, etc.